src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkRepositoryFunction.java

```diff
@@ -328,6 +332,10 @@ private RepositoryDirectoryValue.Builder fetchInternal(
         markerData.put("ENV:" + envKey, clientEnvironment.get(envKey));
       }
 
+      for (Table.Cell<RepositoryName, String, RepositoryName> repoMappings : repoMappingRecorder.recordedEntries().cellSet()) {
+        markerData.put("REPO_MAPPING:" + repoMappings.getRowKey().getName() + "," + repoMappings.getColumnKey(), repoMappings.getValue().getName());
```

Comment:
What happens if you downgrade your Bazel version to something that does not understand this new tag?

Reply:
the marker file version was incremented.

Reply:
Ow, my bad. It indeed was.